spur_controller/nodes/base_controller.py

```diff
@@ -111,7 +111,7 @@ def control(self, sec_idle=1.0):
         v2 = numpy.dot(quaternion_matrix([o.x, o.y, o.z, o.w]), v1)
         self.odom.pose.pose.position.x += v2[0,3]
         self.odom.pose.pose.position.y += v2[1,3]
-        q1 = quaternion_about_axis(self.cmd.angular.z*control_interval, (0, 0, 1))
+        q1 = quaternion_about_axis(-1.0*self.cmd.angular.z*control_interval, (0, 0, 1))
```

Comment:
we shouldn't commit this one
1. it is not confirmed on real robot
2. if it works on real robot, that means we have something wrong around https://github.com/tork-a/spur/pull/31, because on the gazebo, the direction is ok, but there are slightly large drift.